README.md

```diff
@@ -23,43 +24,37 @@ An open source Python based app that replicates the functionality of the existin
 
 ## Get started
 
-####To work with tabcmd, you need to have **Python 3.7+** installed.
-
 To install tabcmd, follow these steps:
+0. To work with tabcmd, you need to have **Python 3.7** or higher installed, and pip
+*see > https://pip.pypa.io/en/stable/installation/ *
+1. Clone the repo and install the app
+> $ git clone https://github.com/tableau/tabcmd.git
+> 
+> $ pip install .
+2. run tests
```

Comment:
Minor - Can we change `run tests` to `(optional) run tests` since people can use tabcmd as a user and they don't need to run tests.